keyboards/ergodox_extended/rules.mk

```diff
@@ -0,0 +1,3 @@
+CUSTOM_MATRIX = yes
```

Comment:
Is a full custom matrix required here? Or would a [lite custom matrix](https://docs.qmk.fm/#/custom_matrix?id=lite) suffice?

Reply:
I'm using a full custom matrix. I could probably get it to work with `lite` but not as implemented.

Reply:
The custom matrix lite API is what all keyboards with a custom matrix scanning implementation will eventually be using. Please change this.

Reply:
Got it switched over to `lite`.